provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -1631,86 +1583,53 @@ func populateRosaClassicClusterState(ctx context.Context, object *cmv1.Cluster,
 		if state.Proxy == nil {
 			state.Proxy = &Proxy{}
 		}
-		state.Proxy.AdditionalTrustBundle = types.String{
-			Value: trustBundle,
-		}
+		state.Proxy.AdditionalTrustBundle = types.StringValue(trustBundle)
 	}
 
 	machineCIDR, ok := object.Network().GetMachineCIDR()
 	if ok {
-		state.MachineCIDR = types.String{
-			Value: machineCIDR,
-		}
+		state.MachineCIDR = types.StringValue(machineCIDR)
 	} else {
-		state.MachineCIDR = types.String{
-			Null: true,
-		}
+		state.MachineCIDR = types.StringNull()
 	}
 	serviceCIDR, ok := object.Network().GetServiceCIDR()
 	if ok {
```

Comment:
Maybe we can have common function that just sets null if value is empty? It seems we have tons of logic like this and can actually save pretty much code lines

Reply:
Maybe we can do it in another PR